canvas_modules/common-canvas/src/common-canvas/svg-canvas-d3.js

```diff
@@ -105,7 +109,18 @@ export default class SVGCanvasD3 {
 		return JSON.parse(JSON.stringify(canvasInfo));
 	}
 
+	onMouseUpdate(e) {
```

Comment:
Please add a function prologue for this function to explain why we need it. I like to write mine starting with a verb. Something like this: "Saves the mouse position for ..."